files/usr/share/cinnamon/applets/inhibit@cinnamon.org/settings-schema.json

```diff
@@ -28,5 +28,14 @@
         "type": "button",
         "description": "Open notification settings",
         "callback": "on_btn_open_system_notification_settings_clicked"
-    }
+    },
+    "section3": {
+        "type": "section",
+        "description": "Startup"
+    },
+    "inhibitPowerAtStartup": {
+		"type": "switch",
+		"default": false,
+		"description": "Inhibit power management at startup"
```

Comment:
Same here except this is mixed spacing of what has been added.

Reply:
Also fixed.